datahub-web-react/src/app/entity/Access/Access.tsx

```diff
@@ -0,0 +1,103 @@
+import { TagOutlined, TagFilled } from '@ant-design/icons';
+import * as React from 'react';
+import styled from 'styled-components';
+import { Role, EntityType, SearchResult } from '../../../types.generated';
+import DefaultPreviewCard from '../../preview/DefaultPreviewCard';
+import { Entity, EntityCapabilityType, IconStyleType, PreviewType } from '../Entity';
+import { getDataForEntityType } from '../shared/containers/profile/utils';
+import { urlEncodeUrn } from '../shared/utils';
+import { SidebarAboutSection } from '../shared/containers/profile/sidebar/AboutSection/SidebarAboutSection';
+import { useGetExternalRoleQuery } from '../../../graphql/accessrole.generated';
+import { EntityProfile } from '../shared/containers/profile/EntityProfile';
+
+const PreviewTagIcon = styled(TagOutlined)`
+    font-size: 20px;
+`;
+// /**
+//  * Definition of the DataHub Access Role entity.
+//  */
+export class RoleEntity implements Entity<Role> {
+    type: EntityType = EntityType.Role;
+
+    icon = (fontSize: number, styleType: IconStyleType, color?: string) => {
+        if (styleType === IconStyleType.TAB_VIEW) {
+            return <TagFilled style={{ fontSize, color }} />;
+        }
+
+        if (styleType === IconStyleType.HIGHLIGHT) {
+            return <TagFilled style={{ fontSize, color: color || '#B37FEB' }} />;
+        }
+
+        return (
+            <TagOutlined
+                style={{
+                    fontSize,
+                    color: color || '#BFBFBF',
+                }}
+            />
+        );
+    };
+
+    isSearchEnabled = () => true;
```

Comment:
we can mark this as "false" - roles will not be searchable _from the main search bar_ -- that's what this determines.

Reply:
unless u were thinking they would be in your instance?

Reply:
we need  searchable from the main search bar i.e why we kept it as true.